data_portal/models/tests/test_labmetadata.py

```diff
@@ -25,3 +26,49 @@ def test_get_labmetadata(self):
             self.assertEqual(get_complete_sequence.library_id, 'L2100001', 'Library result as expected.')
         except ObjectDoesNotExist:
             logger.info(f"Raised ObjectDoesNotExist which is not expected")
+
+    def test_get_by_keyword_not_sequenced(self):
+        # python manage.py test data_portal.models.tests.test_labmetadata.LabMetadataTestCase.test_get_by_keyword_not_requenced
```

Comment:
Typo in `requenced`. And also it is better in docstring wrap, pls.

Tip: In PyCharm, I use shortcut - cursor over function name; `Shift+Alt+CMD C`

```
python manage.py test data_portal.models.tests.test_labmetadata.LabMetadataTestCase.test_get_by_keyword_not_sequenced
```

Reply:
Ah, sorry!
Yes I used PyCharm to get the function name, but then noticed the error in the actual function name and forgot to update the test call ;-)

Reply:
Fixed.